packages/components/src/styles/interactive/interactive.css

```diff
@@ -4,18 +4,21 @@
     outline: revert;
```

Comment:
this should be in another PR as we use squash-merge in our PRs. This would result in one single commit with changes in 2 different components...

As other teams rely on our changelog, we need to be precise here.